tests/installation/sle11_ncc.pm

```diff
@@ -7,11 +7,49 @@ sub run(){
 
     assert_screen 'novell-customer-center', 30;
 
-    # configure later
-    send_key "alt-c";
+    # configure ncc now
+    if (get_var("NCC")) {
+        send_key $cmd{'next'};
+        assert_screen 'ncc-manual-interaction', 60; # contacting server take time
+        send_key 'alt-o';
+        assert_screen 'ncc-security-warning', 30;
+        send_key 'ret';
 
-    sleep 1;
-    send_key $cmd{'next'};
+        my $emailaddr = get_var("NCC_EMAIL");
+        my $activation_code = "";
+        $activation_code = get_var("NCC_SLED_CODE") if (check_var('FLAVOR', 'Desktop-DVD'));
+        $activation_code = get_var("NCC_SLES_CODE") if (check_var('FLAVOR', 'Server-DVD'));
```

Comment:
I think we should rather assign a NCC_CODE to the medium instead of this. Otherwise you will soon enough find yourself having a different code for ppc64 medium or for addon tests, ...